src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -102,8 +100,8 @@
                     </ControlTemplate>
                 </Setter.Value>
             </Setter>
-            <EventSetter Event="Opened"
-                         Handler="OnContextMenuOpened" />
```

Comment:
`Popup.Opened` is not routed event, so it cannot be set here (done in codebehind)